libafl/src/schedulers/testcase_score.rs

```diff
@@ -339,6 +343,10 @@ where
             weight *= 2.0;
         }
 
+        if entry.objectives_found() {
```

Comment:
`objectives_found` sounds like a counter (should it be?)

Reply:
no. it's boolean.
then found_objectives?

Reply:
Renaming is fine, too. But counting doesn't cost performance and maybe it's helpful for some other algorithms?